File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        ans = 0
        for sentence in sentences:
            word_count = 0
            for s in sentence:
                if s == " ":
                    word_count += 1
            if word_count + 1 > ans:
                ans = word_count + 1
        
        return ans

File: 2114-maximum-number-of-words-found-in-sentences/2114-maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int answer = 0;
        for (String sentence: sentences ) {
            String[] words = sentence.split(" ");

            if (words.length > answer) {
                answer = words.length;
            }
        }
        return answer;
    }
}